src/libraries/System.Private.CoreLib/src/System/IO/FileStream.cs

```diff
@@ -130,6 +130,43 @@ public FileStream(string path, FileMode mode, FileAccess access, FileShare share
         }
 
         public FileStream(string path, FileMode mode, FileAccess access, FileShare share, int bufferSize, FileOptions options)
+            : this(path, mode, access, share, bufferSize, options, 0)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="T:System.IO.FileStream" /> class with the specified path, creation mode, read/write and sharing permission, the access other FileStreams can have to the same file, the buffer size,  additional file options and the allocation size.
+        /// </summary>
+        /// <param name="path">A relative or absolute path for the file that the current <see langword="FileStream" /> object will encapsulate.</param>
```

Comment:
`langword` is used for reserved keywords, and `cref` is for API DocIds, so that in MS Docs, the user can click to that API's page.

Also, the full name (which includes namespace) isn't necessary, but just for consistency with the rest of your triple slash comments, this cref can also have the full name:

```suggestion
        /// <param name="path">A relative or absolute path for the file that the current <see cref="System.IO.FileStream" /> instance will encapsulate.</param>
```